data/performance.json

```diff
@@ -128,6 +129,7 @@
     "Cumbria County Council": 5,
     "DBS FIN RCPTS CTS": 1,
     "DBS Mil Pers": 1,
+    "DONCASTER & BASSETLAW TEACHING HOSPITALS NHS FOUNDATION TRUST": 1,
```

Comment:
There is another one with the same name `Doncaster & Bassetlaw Teaching Hospitals NHS Foundation Trust`